bundles/EcommerceFrameworkBundle/Model/AbstractOrderItem.php

```diff
@@ -226,6 +227,16 @@ public function setOrderState($orderState)
         throw new UnsupportedException(__FUNCTION__ . ' is not implemented for ' . get_class($this));
     }
 
+    /**
+     * @throws UnsupportedException
+     *
+     * @param string $comment
+     */
+    public function setComment($comment)
+    {
+        throw new UnsupportedException(__FUNCTION__ . ' is not implemented for ' . get_class($this));
```

Comment:
Again, I'd revert this change for this PR.

Reply:
https://github.com/pimcore/pimcore/pull/7553